administration/src/bp-modules/NavigationBar.tsx

```diff
@@ -14,11 +14,11 @@ const PrintAwareNavbar = styled(Navbar)`
   }
 `
 
-interface Props {
+type Props = {
```

Comment:
I think we should define a proper naming within the project. It not has to be fixed here but i think we should name it <component-name>Props or sth similar

Reply:
You mean this should e.g. be `NavigationBarProps`? Agreed, but I don't think there is a rule for this, we just have to take care of this manually/during reviewing. Do you want me to create an issue for this? Or should we do this on the fly?